core/domain/exp_services.py

```diff
@@ -2003,3 +2008,19 @@ def create_and_save_state_id_mapping_model_for_reverted_exploration(
 
     _save_state_id_mapping(new_state_id_mapping)
     return new_state_id_mapping
+
+
+def delete_state_id_mapping_model_for_exploration(
+        exploration_id, exploration_version):
+    """Removes state id mapping model for the exploration.
+
+    Args:
+        exploration_id: str. Id of the exploration.
+        exploration_version: int. Latest version of the exploration.
+    """
+    state_id_mapping_models = [
```

Comment:
These don't look like models to me. Can we instead call the following?

    exp_models.StateIdMappingModel.delete_state_id_mapping_models(
        exp_id, version_numbers)

Reply:
Done